lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -46,6 +62,9 @@ describe('LMSFilePicker', () => {
         apiCall: fakeApiCall,
       },
       './Dialog': FakeDialog,
+      // Don't mock <FileList> because <NoFiles> requires
+      // it to render for code coverage.
+      './FileList': FileList,
```

Comment:
Some hackery here, because `<NoFiles>` gets passed as a prop, it won't render in this tests (e.g. not full codecov) unless we don't mock `<FileList>`, so this simply forced FileList to render too. 

I also felt `NoFiles` should **continue** being passed as prop and not a child component because it's not a central child component of `FileLIst`.